### ISSUE TITLE
List of links on "All links use a valid format"

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Checklists#838]
Could we take the same approach as with images missing ALT Text?

### Discussed in https://github.com/publishpress/PublishPress-Checklists/discussions/677

<div type='discussions-op-text'>

<sup>Originally posted by **rizaardiyanto1412** April  1, 2024</sup>
When the post is a long post and have many links, and the checklist is failed for item "All links use a valid format", it become tedious to find which links are failing to fulfill that checklist item. 

Can we improve this? Maybe show which links are failing, could be a list or else?

Original request: https://secure.helpscout.net/conversation/2466211002/217916?viewId=6932143</div>

**Comment by rizaardiyanto1412** [comment]
This is already applied here: https://github.com/publishpress/PublishPress-Checklists/pull/829
